packages/access-policy/package.json

```diff
@@ -11,7 +11,7 @@
   "author": "Beyond Essential Systems <admin@tupaia.org> (https://beyondessential.com.au)",
   "main": "dist/index.js",
   "scripts": {
-    "build": "babel src --out-dir dist --source-maps --config-file '../../babel.config.json'",
+    "build": "babel src --delete-dir-on-start --out-dir dist --source-maps --config-file \"../../babel.config.json\"",
```

Comment:
The `delete-dir-on-start` option is annoyingly not documented! I found it while searching around for similar issues. It was added in this PR: https://github.com/babel/babel/pull/6187

I guess we could also simply delete the `dist` file before we run `babel`